dts/msm8916/msm8916-mtp-smb1360.dts

```diff
@@ -12,6 +12,14 @@
 	alcatel-idol347 {
 		model = "Alcatel OneTouch Idol 3 (4.7)";
 		compatible = "alcatel,idol347", "qcom,msm8916", "lk2nd,device";
-		lk2nd,match-cmdline= "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_hx8394d_auo_720p_video:1:none";
+		lk2nd,match-panel;
+
+		panel {
+			compatible = "alcatel,idol347-panel";
+
+			qcom,mdss_dsi_hx8394d_auo_720p_video {
+				compatible = "alcatel,auo-hx8394d";
+			};
```

Comment:
Please add all possible panels here (even if you don't support them yet). Then we don't need a new lk2nd release later.